owltorrent-network/src/main/java/edu/rice/owltorrent/network/messagereader/BusyWaitMessageReader.java

```diff
@@ -0,0 +1,60 @@
+package edu.rice.owltorrent.network.messagereader;
```

Comment:
This is not actually a busy wait reader, more accurately a blocking reader (possible I am mistaken)

Reply:
Yeah you are correct.